scss/modules/_breadcrumbs.scss

```diff
@@ -0,0 +1,189 @@
+////
+/// @author       Web Team at Canonical Ltd
+/// @link         http://ubuntudesign.github.io/vanilla-framework/docs/#mixin-vf-lists
+/// @since        0.0.3
+////
+
+/// Default blockquote styling
+/// @group Blockquotes
+/// @example
+/// <blockquote>
+///   ...
+/// </blockquote>
+@mixin vf-breadcrumbs {
+  // scss-lint:disable NestingDepth
```

Comment:
Can we reduce the covering of this lint rule disable please. We should only wrap the exact exception.